tibanna/cw_utils.py

```diff
@@ -324,19 +324,117 @@ def max_ebs_read_used_all_pts(self):
         pts = [(r['Average'], r['Timestamp']) for r in res['Datapoints']]
         return[p[0] for p in sorted(pts, key=lambda x: x[1])]
 
+    
+    @staticmethod
+    def extract_metrics_data(file_contents):
+      # This function takes a file contents string and parses it column wise.
+      # It returns the header and the data in a way that can be injected into JS code
+      # `data` is a Python dict, that contains all that data for later processing
+      # outside of this function.
+      columns = []
+      columns_js = "[]"
+      data = {}
+      data_js = "[]"
```

Comment:
FYI, you can declare multiple variables on the same line for a more compact looking declaration overall:
```
columns, data = [], {}
columns_js, data_js = "[]", "[]"
```